api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -33,13 +40,15 @@
 api_logger.addHandler(logging.StreamHandler())
```

Comment:
Not sure - I see that Andrew added those. Let's ask him when he's back.

Reply:
Not sure if it's good hygiene to put that in __init__.py directly, but maybe into a new logging.py module that is _imported_ by __init__.py.  Either way, out-of-scope change for this PR, but worth considering (I think we have a pending "api logging" issue where it could be addressed).